### PR TITLE
Blog-app-rails: Create data model

### PR DESCRIPTION
**Pull request by yodit93**
### In this project, I

- [ ] created a new Rails blog app,
- [ ] setup Postgres database,
- [ ] built the project schema,
- [ ] created and run the migration files,

- user
- post
- comment
- like

- [ ] included foreign keys,
- [ ] created index for all columns that are foreign keys.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem 'rails', '~> 7.0.5'
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails'
+
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+
```
Comment (mwanawabangona) [inline Gemfile]: - [ ] `Required`
- [ ] Amazing job so far! 🥂 
- [ ] However, you are required to use `postgresql` and not `sql lite` 😞 
- [ ] please use the correct database 🥂 
- [ ] This [article](https://medium.com/geekculture/ruby-on-rails-switch-from-sqlite3-to-postgres-590009645c25) may help if you will have issues

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @yodit93,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Gemfile (hunk 1 above)
Comment (yodit93) [reply Gemfile]: I will. Thank you for the suggesion.

**Review by rustam-tolipov — APPROVED** [review]
Hi @yodit93, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well implemented code based on the previous reviewer's request 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yodit93** [comment]
Thank you!!

**Pull request merged**